app/assets/javascripts/components/sprints/Sprint.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import classname from 'classnames';
+import PropTypes from "prop-types";
+import Stories from "../stories/Stories";
+import SprintHeader from './SprintHeader'
+import { sprintPropTypesShape } from '../../models/beta/iteration';
+import { pastIterationPropTypesShape } from '../../models/beta/pastIteration';
+
+const propTypes = {
+  fetchStories: PropTypes.func,
+  sprint: PropTypes.oneOf([
+    sprintPropTypesShape,
+    pastIterationPropTypesShape
+  ])
+};
+
+const defaultProps = {
+  fetchStories: undefined,
+  sprint: {
+    number: 0,
+    startDate: 0,
+    points: 0,
+    stories: []
+  }
+};
+
+const Sprint = ({
+  fetchStories,
+  sprint
+}) => {
+  const isDone = () => sprint.hasOwnProperty('hasStories');
```

Comment:
You can use [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) here:
```js
const isDone = useMemo(() => sprint.hasOwnProperty('hasStories'), [sprint]);

isDone // true|false
```

Reply:
Gotcha! Thanks.